iree/compiler/Codegen/Common/LinalgBufferizePass.cpp

```diff
@@ -1378,7 +1378,10 @@ static LogicalResult convertVectorTransferWriteOp(OpBuilder &b,
   if (!resultType) return success();
   Value resultBuffer = bvm.lookup(result);
 
-  if (!plan.isEquivalent(op.source(), result)) {
+  if (!plan.isEquivalent(op.source(), result) &&
```

Comment:
This is fine for now, but a better approach is to just un-cse the `init_tensor` ops so that they are different undef values.